src/module.c

```diff
@@ -3719,6 +3719,14 @@ ValkeyModuleString *VM_GetClientUserNameById(ValkeyModuleCtx *ctx, uint64_t id)
     return str;
 }
 
+/* Returns 1 if commands are arriving from the primary client or AOF client
+ * and should never be rejected.
+ * Returns 0 otherwise (and also if ctx or if the client is NULL). */
```

Comment:
Since this documentation is used to generate the public facing docs, I think it would also be useful to document *why* one would use this. Specifically it should be used for validation that might cause divergence on primaries and replicas (or AOF files).

Reply:
Yes, that makes sense. I updated the comment